core/controllers/profile_test.py

```diff
@@ -1324,9 +1324,11 @@ def test_login_url_gets_created_for_signed_out_users(self):
         self.assertTrue(response['login_url'].endswith('random_url'))
 
     def test_invalid_input_exception(self):
-        response = self.get_json('/url_handler', expected_status_int=400)
+        response = self.get_json(
+                        '/url_handler', expected_status_int=400
```

Comment:
The indentation here and in the next line seems wrong.

I wonder why you needed to change this, anyway?

Reply:
Hey @seanlip I have committed the change by changing the indentation. Can you please verify these changes ?
Thank you

And I have made these changes because since I have added `current_url` as an argument in "HANDLER_ARGS_SCHEMAS". So, we are getting a new error message which is `Missing key in handler args: current_url.`